src/Parser/Core/Modules/ManaValues.js

```diff
@@ -0,0 +1,36 @@
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+
+const MANA_CLASS_RESOURCE_ID = 0;
+
+class ManaValue extends Module {
+  lowestMana = 1100000;
+  endingMana = 0;
+
+  on_byPlayer_cast(event) {
+    // class resource type 0 means the resource is mana
+    event.classResources.forEach(classResource => {
```

Comment:
This crashes when the event is a melee attack (auto attack), please check that `event.classResources` is not falsy